### PR TITLE
Added a link to Feedback site at the error messages

### PR DESCRIPTION
**Pull request by LW001**
yw

### CODE REVIEW

File: Commands/Commands/queue.js [hunk 1]
```diff
@@ -325,7 +325,7 @@ commands.registerVote = {
                     message: (e.message !== undefined) ? e.message : JSON.stringify(e)
                   }, e).catch(e => {
                     if (e === 'Not found') {
-                      bot.Channels.get(doc.channel).sendMessage(`${user.mention}, your details are not found.`).then(errmsg => {
+                      bot.Channels.get(doc.channel).sendMessage(`${user.mention}, your details were not found. Make sure you've logged into the website at <https://feedback.discordapp.com> at least once.`).then(errmsg => {
```
Comment (Arzte) [inline Commands/Commands/queue.js]: Why is it in brackets?

Comment (Dougley) [reply Commands/Commands/queue.js]: Embed escape

Comment (Arzte) [reply Commands/Commands/queue.js]: ahh.

File: Commands/Commands/queue.js [hunk 2]
```diff
@@ -325,7 +325,7 @@ commands.registerVote = {
                     message: (e.message !== undefined) ? e.message : JSON.stringify(e)
                   }, e).catch(e => {
                     if (e === 'Not found') {
-                      bot.Channels.get(doc.channel).sendMessage(`${user.mention}, your details are not found.`).then(errmsg => {
+                      bot.Channels.get(doc.channel).sendMessage(`${user.mention}, your details were not found. Make sure you've logged into the website at <https://` + config.uservoice.subdomain + `.` + config.uservoice.domain + `> at least once.`).then(errmsg => {
```
Comment (ThaTiemsz) [inline Commands/Commands/queue.js]:
```js
bot.Channels.get(doc.channel).sendMessage(`${user.mention}, your details were not found. Make sure you've logged into the website at <https://${config.uservoice.subdomain}.${config.uservoice.domain}> at least once.`).then(errmsg => {
```

File: Commands/Commands/uservoice.js [hunk 3]
```diff
@@ -55,7 +55,7 @@ commands.vote = {
       })
     }).catch(e => {
       if (e === 'Not found') {
-        msg.reply("I was unable to find your details, make sure you've logged into the website at least once.").then(errmsg => {
+        msg.reply("I was unable to find your details, make sure you've logged into the website at <https://`+ config.uservoice.subdomain + `.` + config.uservoice.domain + `> at least once.").then(errmsg => {
```
Comment (ThaTiemsz) [inline Commands/Commands/uservoice.js]:
```js
msg.reply(`I was unable to find your details, make sure you've logged into the website at <https://${config.uservoice.subdomain}.${config.uservoice.domain}> at least once.`).then(errmsg => {
```

File: Commands/Commands/uservoice.js [hunk 4]
```diff
@@ -134,7 +134,7 @@ commands.submit = {
         })
       }).catch(e => {
         if (e === 'Not found') {
-          msg.reply("I was unable to find your details, make sure you've logged into the website at least once.").then(errmsg => {
+          msg.reply("I was unable to find your details, make sure you've logged into the website at <https://`+ config.uservoice.subdomain + `.` + config.uservoice.domain + `> at least once.").then(errmsg => {
```
Comment (ThaTiemsz) [inline Commands/Commands/uservoice.js]:
```js
msg.reply(`I was unable to find your details, make sure you've logged into the website at <https://${config.uservoice.subdomain}.${config.uservoice.domain}> at least once.`).then(errmsg => {
```

File: Commands/Commands/uservoice.js [hunk 5]
```diff
@@ -225,7 +225,7 @@ commands.comment = {
       })
     }).catch(e => {
       if (e === 'Not found') {
-        msg.reply("I was unable to find your details, make sure you've logged into the website at least once.").then(errmsg => {
+        msg.reply("I was unable to find your details, make sure you've logged into the website at <https://`+ config.uservoice.subdomain + `.` + config.uservoice.domain + `> at least once.").then(errmsg => {
```
Comment (ThaTiemsz) [inline Commands/Commands/uservoice.js]:
```js
msg.reply(`I was unable to find your details, make sure you've logged into the website at <https://${config.uservoice.subdomain}.${config.uservoice.domain}> at least once.`).then(errmsg => {
```

**Review by ThaTiemsz — CHANGES_REQUESTED** [review]

File: Commands/Commands/uservoice.js (hunk 5 above)
Comment (LW001) [reply Commands/Commands/uservoice.js]: Had all of these before, though Travis didn't like it... https://github.com/SteamingMutt/DiscordFeedback/pull/42/commits/d7393a36458ab7f1919ac2bd40e88dc0301c1160

Comment (ThaTiemsz) [inline Commands/Commands/uservoice.js]: @nullpixel Fix travis

Comment (Dougley) [reply Commands/Commands/uservoice.js]: @LW001 You tried to use template syntax outside of template literal strings, Travis is therefore correct in erroring out.
Use the spoon-feeding @ThaTiemsz gave you.

**Review by Dougley — CHANGES_REQUESTED** [review]
Use template literals.

File: Commands/Commands/queue.js [hunk 6]
```diff
@@ -414,7 +414,7 @@ commands.registerVote = {
                     message: (e.message !== undefined) ? e.message : JSON.stringify(e)
                   }, e).catch(e => {
                     if (e === 'Not found') {
-                      bot.Channels.get(config.discord.feedChannel).sendMessage(`${user.mention}, your details are not found.`).then(errmsg => {
+                      bot.Channels.get(config.discord.feedChannel).sendMessage(`${user.mention}, your details were not found. Make sure you've logged into the website at <https://` + config.uservoice.subdomain + `.` + config.uservoice.domain + `> at least once.`).then(errmsg => {
```
Comment (Dougley) [inline Commands/Commands/queue.js]: Missed one

**Review by Dougley — CHANGES_REQUESTED** [review]

**Review by Dougley — APPROVED** [review]

**Pull request merged**